Library/Stubs/Generator.php

```diff
@@ -99,17 +99,27 @@ public function generate(string $namespace, string $path, string $indent)
      *
      * @return string
      */
-    protected function buildClass(ClassDefinition $class, $indent)
+    protected function buildClass(ClassDefinition $class, string $indent): string
     {
-        $source = <<<EOF
-<?php
+        $source = "<?php\n\n";
```

Comment:
```suggestion
        $source = "<?php" . PHP_EOL . PHP_EOL;
```

As you use `PHP_EOL` in all places...

Reply:
Looks better :) I'll change this